base/range.jl

```diff
@@ -59,6 +59,9 @@ automatically such that there are `length` linearly spaced elements in the range
 If `step` and `stop` are provided and `length` is not, the overall range length will be computed
 automatically such that the elements are `step` spaced.
 
+Special care is taken to ensure intermediate values are computed rationally.
+For a simpler linearly spaced range-like object with less overhead, see [`LinRange`](@ref).
```

Comment:
```suggestion
For a construction with less overhead, see [`LinRange`](@ref).
```
(The previous sentence implied `LinRange` is not a range but "range-like"; also, "linearly spaced" is redundant to ranges...)

Reply:
I made a change with slightly different wording than you suggested.

Reply:
Great